data/json/items/melee.json

```diff
@@ -387,8 +369,7 @@
         "techniques" : ["WBLOCK_1"],
         "volume" : 5,
         "bashing" : 4,
-        "price" : 2400,
-        "price_postapoc": 0
```

Comment:
For things like brooms and two-by-fours it doesn't really make sense that people would keep buying them when they can just smash nearby furniture. They're worth as much as rocks or scrap metal.